app/styles/ui/changes/_changes-list.scss

```
@@ -276,3 +282,74 @@
     display: inline;
   }
 }
+
+.no-changes-filtered {
+  max-width: 365px;
+  min-width: 180px;
+  width: 100%;
+  margin: var(--spacing-half) auto;
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  padding: var(--spacing);
+  box-sizing: border-box;
+  overflow: hidden; // Prevent scrollbars
```

```suggestion
  overflow-x: hidden;
  overflow-y: auto; 
```

:D We want scrollbars. Even with the nice, hide image on small screens that you added, we are still clipping the content of the no results message if there are multiple filters applied. Additionally, if a user has the screen applies the zoom feature, the content is clipped. Thus, we do want the scrollbars in this case.

---

Great catch! You're right - we should allow vertical scrolling for accessibility and zoom scenarios. I'll update this.